site/en/blog/user-agent-reduction-deprecation-trial/index.md

```diff
@@ -39,23 +39,23 @@ Sites should still audit their usage of the User-Agent header and related APIs,
 
 ### Register for the trial
 
-To register for the origin trial and get a token for your domains, visit the [User Agent Reduction deprecation trial page](/origintrials/#/view_trial/2608710084154359809).
+To register for the origin trial and get a token for your domains, visit the [User Agent Reduction deprecation trial page](/origintrials/#/view_trial/2608710084154359809). If you are a third-party registering please check 'Third-party matching'.
 
 ### Setup
 
 Once you've registered for the trial, update your HTTP response headers with the following:
 
 1.  Add `Origin-Trial: <ORIGIN TRIAL TOKEN>` to your HTTP response header. <`ORIGIN TRIAL TOKEN`> contains the token you got when registering for the origin trial.
 1.  Add `Accept-CH: Sec-CH-UA-Full` to your HTTP response header. Setting `Accept-CH` will only cause the full User-Agent string to be sent on subsequent requests to the origin.
-1. To resend the first navigation request with the full User-Agent string, add `Critical-CH: Sec-CH-UA-Full` to your HTTP response header, in addition to the `Accept-CH` and `Origin-Trial` headers.
-1.  If you have opted in your site to the trial and also want third-party subresource requests to also receive the full UA string, you have two options:
-    - Add a `Permissions-Policy` header with the third-party domains that should receive the full UA.
-        -  To allow a named list of third-party domains, add `Permissions-Policy: ch-ua-full=(self "https://thirdparty.example.com")`.
+1.  If the full User-Agent string is critical on first request add `Critical-CH: Sec-CH-UA-Full` to your HTTP response header, in addition to the `Accept-CH` and `Origin-Trial` headers.
+1.  For participants joining the reverse origin trial we suggest allowing all third-party domains access to the full User-Agent string. Failure to extend third-party domains access to the full User-Agent string will block their full User-Agent string access regardless of their own reverse origin trial registration. You can allow full User-Agent string access to third-party domains by one of the following two options:
```

Comment:
nit: the preferred term is deprecation trial, rather than "reverse origin trial".

Reply:
Good catch, I'll fix that.